extension/spark-doris-connector/src/main/thrift/doris/DorisExternalService.thrift

```diff
@@ -56,6 +56,8 @@ struct TScanOpenParams {
   11: optional i16 keep_alive_min
 
   12: optional i32 query_timeout
+
+  13: optional i64 mem_limit
```

Comment:
Plz add comment for this field. Or to change it to a more explicit name, eg: `query_mem_limit`

Reply:
Done